.github/workflows/release.yml

```diff
@@ -19,8 +19,24 @@ jobs:
         uses: actions-rust-lang/setup-rust-toolchain@v1.3.7
         env:
           RUST_CACHE_KEY_OS: rust-cache-${{ matrix.arch }}
+      - name: Determine Release Info
+        id: info
+        env:
+          GITHUB_REF: ${{ github.ref }}
+        run: |
+          VERSION=${GITHUB_REF##*v}
+          MAJOR=${VERSION%%.*}
+          MINOR=${VERSION%.*}
+          MINOR=${MINOR#*.}
+          PATCH=${VERSION##*.}
+          echo "VERSION=${VERSION}" >> $GITHUB_ENV
+          echo "OUTPUTDIR=youki_${MAJOR}_${MINOR}_${PATCH}_linux" >> $GITHUB_ENV
+          echo "INNERDIR=youki-${VERSION}" >> $GITHUB_ENV
       - name: Install just
         uses: taiki-e/install-action@just
+      - name: Version update
+        run: |
+          sudo env PATH=$PATH just version-up ${{ env.VERSION }}
```

Comment:
Before running release CI, we need to update a version in `Cargo.toml`

Reply:
> Before running release CI, we need to update a version in `Cargo.toml`

Wouldn't it be better to modify it when publishing? Do we need to submit a separate PR to modify the version number?

Reply:
Who will version Cargo.toml after release? For example, create a tag v2.0.0 and push. But then the tag's branch, Cargo.toml, is not 2.0.0, is it?

Reply:
Yes, the branch is not, but the corresponding file has been changed, and pr will be automatically submitted to modify these versions later.

Reply:
I feel that the branch needs to be in a released state for later reference.

Reply:
Can't we just create a PR for the release and create a tag once it is merged?

Reply:
Of course I'd like this feature! However, I feel a little problem with the flow. The concept itself is good.

Reply:
> Can't we just create a PR for the release and create a tag once it is merged?

Indeed, it‘s better to adjust the process. Just do a manual workflow.